assets/scss/core/_reset.scss

```diff
@@ -0,0 +1,351 @@
+/*
+
+RESET DOPPLER STYLE GUIDES
```

Comment:
this is generated? or copy from internet?

Reply:
Agregue un normalize que no tenia y lo renombre para que podamos poner los posibles reset que hagamos. Si prefieren que lo elimine y vayamos agregando uno a uno, no hay problema.